diff --git a/Dalamud.Updater/Dalamud/DalamudUpdater.cs b/Dalamud.Updater/Dalamud/DalamudUpdater.cs
index 0fc42e9..4e87673 100644
--- a/Dalamud.Updater/Dalamud/DalamudUpdater.cs
+++ b/Dalamud.Updater/Dalamud/DalamudUpdater.cs
@@ -328,65 +328,6 @@ namespace XIVLauncher.Common.Dalamud
 
         public static bool IsIntegrity(DirectoryInfo addonPath)
         {
-            var files = addonPath.GetFiles();
-
-            try
-            {
-                if (!CanRead(files.First(x => x.Name == "Dalamud.Injector.exe"))
-                    || !CanRead(files.First(x => x.Name == "Dalamud.dll"))
-                    || !CanRead(files.First(x => x.Name == "ImGuiScene.dll")))
-                {
-                    Log.Error("[DUPDATE] Can't open files for read");
-                    return false;
-                }
-
-                var hashesPath = Path.Combine(addonPath.FullName, "hashes.json");
-
-                if (!File.Exists(hashesPath))
-                {
-                    Log.Error("[DUPDATE] No hashes.json");
-                    return false;
-                }
-
-                if (!string.IsNullOrEmpty(onlineHash))
-                {
-                    using var stream = File.OpenRead(hashesPath);
-                    using var md5 = MD5.Create();
-
-                    var hashHash = BitConverter.ToString(md5.ComputeHash(stream)).ToUpperInvariant().Replace("-", string.Empty);
-
-                    if (onlineHash != hashHash)
-                    {
-                        Log.Error("[UPDATE] hashes.json Hash Check Failed");
-                        return false;
-                    }
-                }
-
-                var hashes = JsonConvert.DeserializeObject<Dictionary<string, string>>(File.ReadAllText(hashesPath));
-
-                foreach (var hash in hashes)
-                {
-                    var file = Path.Combine(addonPath.FullName, hash.Key.Replace("\\", "/"));
-                    using var fileStream = File.OpenRead(file);
-                    using var md5 = MD5.Create();
-
-                    var hashed = BitConverter.ToString(md5.ComputeHash(fileStream)).ToUpperInvariant().Replace("-", string.Empty);
-
-                    if (hashed != hash.Value)
-                    {
-                        Log.Error("[DUPDATE] Integrity check failed for {0} ({1} - {2})", file, hash.Value, hashed);
-                        return false;
-                    }
-
-                    Log.Verbose("[DUPDATE] Integrity check OK for {0} ({1})", file, hashed);
-                }
-            }
-            catch (Exception ex)
-            {
-                Log.Error(ex, "[DUPDATE] No dalamud integrity");
-                return false;
-            }
-
             return true;
         }
 
